Makefile

```diff
@@ -108,7 +108,7 @@ dkms_modules_uninstall: dkms_check
 	dkms status $(DKMS_MODULES_NAME) | \
 	    while IFS=':' read -r modules status; do \
 	        echo "$$modules" | { \
-	            IFS=', ' read -r modules_name modules_version \
```

Comment:
Thanks! I'm curious, though, where does that change come from? Has `dkms status` output format changed?